kerl

```diff
@@ -2099,6 +2099,39 @@ get_active_path() {
     fi
 }
 
+fix_otp_installations() {
+    if [ -f "$KERL_BASE_DIR"/otp_installations ]; then
+        inexisting_names=""
+        while read -r l; do
+            names=$(echo "$l" | cut -d' ' -f1)
+            path=$(echo "$l" | cut -d' ' -f2)
+            if [ ! -e "$path" ]; then
+                inexisting_names="$path $inexisting_names"
+            fi
+        done <"$KERL_BASE_DIR"/otp_installations
+        for inexisting_name in ${inexisting_names}; do
+            escaped="$(echo "$inexisting_name" | \sed "$SED_OPT" -e 's#/$##' -e 's#\/#\\\/#g')"
```

Comment:
I wonder if this should be a function since it's copy-paste from some below (the `sed` bit).

Reply:
Resolve'ing as preparing to merge.